CribbageTournamentTracker/src/components/Header.svelte

```diff
@@ -27,13 +31,45 @@
 				<span class="navbar-toggler-icon" />
 			</button>
 		</div>
-		<div class="collapse navbar-collapse" id="navbarNav">
+		<!--Navigation icons-->
+		<div class="collapse navbar-collapse justify-content-end w-100" id="navbarNav">
+			<form class="d-flex" role="search">
+				<input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" />
+				<button class="btn btn-outline-success me-2" type="submit">Search</button>
+			</form>
 			<ul class="navbar-nav">
 				<li class="nav-item">
-					<a class="nav-link active" aria-current="page" href="/">Home</a>
+					<a class="nav-link active" href="/">
+						<img
+							src={BellIcon}
+							alt="Bell Logo"
+							width="30"
+							height="24"
+							class="d-inline-block pe-2"
```

Comment:
Is there a better way to code these images rather than having them with a static width/height?